misc/python/materialize/mzcompose/__init__.py

```diff
@@ -69,6 +69,7 @@
     "persist_streaming_snapshot_and_fetch_enabled": "true",
     "enable_unified_clusters": "true",
     "enable_jemalloc_profiling": "true",
+    "enable_comment": "true",
```

Comment:
should also add `"enable_sink_doc_on_option": "true"` here so we can use it easily in testing.

Reply:
done!